code-challenges.js

```diff
@@ -35,35 +58,100 @@ const temperature3 = 212
 
 // Pseudo code:
 
+  // Create a function that takes two arrays as parameters
+
+  // Combine the arrays using the concat() method
+
+  // Get the length of the combined array using the length
+
+  //  Return the length of the combined array.
+
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+const baseball =  padres1984WorldSeriesRuns.concat(padres1988WorldSeriesRuns).length;
+console.log
+
+
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+  
+  //Create a function that takes in a string parameter.
+
+  // Declare a variable to hold the reversed string and assign an empty string to it.
+
+  // Using a for loop, start at the end of the input string and iterate backwards until the beginning.
+ 
+  // With each iteration, add the current character to the reversed string variable.
+
+  // Return the reversed string variable.
 
 const currentCohort = "Delta 2023"
 // Expected output: "3202 atleD"
 
+const reverseString = (str) => {
+  let reversedString = "";
+  for (let i = str.length - 1; i >= 0; i--) {
+    reversedString += str[i];
+  }
+  return reversedString;
+};
+
+const currentCohort = "Delta 2023";
+const reversed = reverseString(currentCohort);
+console.log(reversed);
+
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
 
-const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+// Define the array as array and the given value as value
 
-const givenValue1 = 42
-// Expected output: 7
+//Find the last index of the given value in the array 
+
+//Output the lastIndex as the final result
+
+const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10];
+
+const givenValue1 = 42;
+console.log(numberOfConnections.lastIndexOf(givenValue1)); // Output: 7
+
+const givenValue2 = 10;
+console.log(numberOfConnections.lastIndexOf(givenValue2)); // Output: 8
 
-const givenValue2 = 10
-// Expected output: 8
 
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
-// Pseudo code:
+//Pseudo code:
+
+//Define the arrays sanDiegoSummerTemperatures and sanDiegoWinterTemperatures
+
+//Sort each array in descending order, arranging the numbers from largest to smallest
```

Comment:
Great job using the sort method to arrange in the order you need. You should include the function you are passing explaining the a, b in your pseudocode.

Reply:
okay, I'll remember to do that in the future.